--- zdump_save.c	2017-11-19 09:39:52.255863414 +0000
+++ zdump.c	2017-11-19 09:40:00.376451760 +0000
@@ -795,38 +795,6 @@
 		abbrok(abbr(tmp), zone);
 }
 
-#if !HAVE_SNPRINTF
-# include <stdarg.h>
-
-/* A substitute for snprintf that is good enough for zdump.  */
-static int
-snprintf(char *s, size_t size, char const *format, ...)
-{
-  int n;
-  va_list args;
-  char const *arg;
-  size_t arglen, slen;
-  char buf[1024];
-  va_start(args, format);
-  if (strcmp(format, "%s") == 0) {
-    arg = va_arg(args, char const *);
-    arglen = strlen(arg);
-  } else {
-    n = vsprintf(buf, format, args);
-    if (n < 0)
-      return n;
-    arg = buf;
-    arglen = n;
-  }
-  slen = arglen < size ? arglen : size - 1;
-  memcpy(s, arg, slen);
-  s[slen] = '\0';
-  n = arglen <= INT_MAX ? arglen : -1;
-  va_end(args);
-  return n;
-}
-#endif
-
 /* Store into BUF, of size SIZE, a formatted local time taken from *TM.
    Use ISO 8601 format +HH:MM:SS.  Omit :SS if SS is zero, and omit
    :MM too if MM is also zero.
